1.8/usage/cli/update.md

```diff
@@ -12,11 +12,10 @@ You can update the DC/OS CLI to the latest version or downgrade to an older vers
 
 You can upgrade an existing DC/OS CLI installation to the latest build.
 
-1.  From your DC/OS CLI installation directory, enter this command to update the DC/OS CLI:
+1.  From your DC/OS CLI installation directory, enter this command to update the DC/OS CLI with the version (`<version>`) specified:
```

Comment:
1. remove the current CLI binary: `rm path/to/binary/dcos`
2. From the directory you want to install the new DC/OS CLI binary, run: